benefits/core/templates/core/includes/form.html

```diff
@@ -7,70 +7,57 @@
+                  data-action="submit">
+            {{ form.submit_value }}
+          </button>
         {% else %}
-          <div class="form-group">
-          {% endif %}
+          <button class="btn btn-lg btn-primary">{{ form.submit_value }}</button>
+        {% endif %}
+      </div>
+    {% endif %}
 
-          {% if field.label %}
-            <label for="{{ field.id_for_label }}" class="form-control-label h2">
-              {{ field.label }}
-              {% if field.field.required %}<span class="required-label">*</span>{% endif %}
-            </label>
-          {% endif %}
-
-          {{ field }}
-
-          {% if field.help_text %}<small class="form-text text-muted">{{ field.help_text }}</small>{% endif %}
-
-          {% if field.errors %}
-            <div class="error-message">
-              {% for error in field.errors %}<p>{{ error }}</p>{% endfor %}
-            </div>
-          {% endif %}
-        </div>
-      {% endfor %}
-
-      {% if form.submit_value %}
-        <div class="form-group">
-          {% if recaptcha.enabled %}
-            <button class="btn btn-lg btn-primary g-recaptcha"
-                    data-sitekey="{{ recaptcha.site_key }}"
-                    data-callback="onSubmit"
-                    data-action="submit">
-              {{ form.submit_value }}
-            </button>
-          {% else %}
-            <button class="btn btn-lg btn-primary">{{ form.submit_value }}</button>
-          {% endif %}
-        </div>
-      {% endif %}
-
-      {% if recaptcha.enabled %}
-        <script src="{{ recaptcha.api_url }}"></script>
-        <script>
-                function onSubmit(token) {
-                    $("form[action='{{form_action}}']").submit();
-                }
-        </script>
-      {% endif %}
+    {% if recaptcha.enabled %}
+      <script src="{{ recaptcha.api_url }}"></script>
       <script>
-            $(function(){
-                $("form[action='{{form_action}}']").on("submit", function(e) {
-                    $(".form-group", this).removeClass("with-errors");
-                    $(".error-message", this).remove();
-
-                    if ("{{ form.submitting_value }}" !== "") {
-                        $("button[data-action='submit']", this)
-                            .attr("role", "status")
-                            .attr("disabled", "true")
-                            .text("{{ form.submitting_value }}")
-                            .addClass("disabled");
-                    }
-                });
-            });
+              function onSubmit(token) {
+                  $("form[action='{{form_action}}']").submit();
+              }
       </script>
-    </form>
-
-  {% endif %}
+    {% endif %}
+
+    <script>
+          $(function(){
+              $("form[action='{{form_action}}']").on("submit", function(e) {
+                  if ("{{ form.submitting_value }}" !== "") {
+                      $("button[data-action='submit']", this)
```

Comment:
I don't think this is a regression from this PR, but I just noticed... not all of the `button`s on this template have the `data-action` attribute, so this selector would not find it sometimes

Reply:
yea sorry the rebase commits https://github.com/cal-itp/benefits/commit/4413ea0ec6f4e40f5834450cfcfb6d917c7d5bc3 are all stemming from difficulty of managing several open PRs + rebasing dev + slow GitHub Actions. and yes, the spacing on this form template is really confusing.

Reply:
I can take care of and test this Form button issue in either https://github.com/cal-itp/benefits/issues/947 and https://github.com/cal-itp/benefits/issues/972. The Button CSS code, Button viewmodel and Form HTML code have not been changed yet in this Sprint and this is the first of many PRs to start cleaning this all up.

Reply:
Is this actually the cause of https://github.com/cal-itp/benefits/issues/948?

Reply:
@angela-tran probably